deltacat/constants.py

```diff
@@ -37,6 +38,15 @@
     False,
 )
 
+# CLI Args
+METAFILE_FORMAT_KEY = "metafile-format"
```

Comment:
Prefer to use standard environment variable casing here:
```suggestion
METAFILE_FORMAT_KEY = "METAFILE_FORMAT"
```